dist/tools/backport_pr/backport_pr.py

```diff
@@ -135,6 +135,16 @@ def main():
     if status != 200:
         print("Could not retrieve user: {}".format(user['message']))
         exit(1)
+    scopes = dict(g.getheaders())
+    if 'X-OAuth-Scopes' in scopes:
+        scopes = scopes['X-OAuth-Scopes']
+    else:
+        scopes = scopes['x-oauth-scopes']
```

Comment:
Could you please refactor your first use of `scopes` here and not reuse the variable first for the generic headers (from `g.getheaders()`) and then for the actual token `scopes`?

I'd prefer if you rename the first use of `scopes` here to something else as the result from `g.getheaders()` is a bit more than just the token scopes.

Reply:
i will